File: app/src/main/java/com/vkpapps/thunder/model/VideoInfo.java
package com.vkpapps.thunder.model;
/***
 * @author VIJAY PATIDAR
 */
public class VideoInfo {
    private String name,path;
    private boolean selected;
    public VideoInfo(String name, String path) {
        this.name = name;
        this.path = path;
    }

    public String getName() {
        return name;
    }

    public boolean isSelected() {
        return selected;
    }

    public void setSelected(boolean selected) {
        this.selected = selected;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPath() {
        return path;
    }

    public void setPath(String path) {
        this.path = path;
    }
}

File: app/src/main/java/com/vkpapps/thunder/model/VideoInfo.kt
package com.vkpapps.thunder.model

import android.net.Uri
import androidx.annotation.NonNull
import androidx.room.Entity
import androidx.room.PrimaryKey

/***
 * @author VIJAY PATIDAR
 */
@Entity
class VideoInfo(var name: String, var uri: Uri, val size: Long, val lastModified: Long) {
    @PrimaryKey
    @NonNull
    lateinit var id: String
    var isSelected = false
}